net/ops/convolution.py

```diff
@@ -85,9 +86,9 @@ def batchnorm(self, layer, inp):
                 'updates_collections' : None,
                 'is_training': layer.h['is_training']
                 })
-            v = tf.__version__.split('.')[1]
-            if int(v) < 12: key = 'initializers'
-            else: key = 'param_initializers'
+
+            v = tf.__version__.split('.')
+            key = 'param_initializers'
```

Comment:
`v = tf.__version__.split('.')` is no longer needed.
`{'param_initializers' : layer.w}` should be put directly inside the first definition of `args` .